plenum/cli/cli.py

```diff
@@ -911,7 +911,7 @@ def newNode(self, nodeName: str):
                 self.bootstrapKey(self.activeWallet, node)
 
             for identifier, verkey in self.externalClientKeys.items():
-                node.clientAuthNr.addClient(identifier, verkey)
+                node.clientAuthNr.addIdr(identifier, verkey)
```

Comment:
We use the word `Idr` to mean identifier in the codebase a lot, so its going to be clear, also the documentation says it.